packages/cascara/src/ui/Form/Form.js

```diff
@@ -9,6 +9,9 @@ import ModuleError from '../../modules/ModuleError';
 import { actionModules, dataModules } from '../../modules/ModuleKeys';
 import { formActionModules, formModules } from './modules';
 import ActionBar from './components/ActionBar';
+import FormLoading from './FormLoading';
+import FormEmpty from './FormEmpty';
```

Comment:
Can we rename these to TemporaryLoading, TemporaryEmpty and set these on both Form and replace the ones on Table? 🙏🏽

Reply:
New files created